akka-persistence-tck/src/main/scala/akka/persistence/CapabilityFlags.scala

```diff
@@ -52,3 +52,9 @@ trait SnapshotStoreCapabilityFlags extends CapabilityFlags {
   // no flags currently
 }
 //#snapshot-store-flags
+
+trait CurrentEventsByTagCapabilityFlags extends CapabilityFlags {
```

Comment:
by the way, how is the ordering and the offset releated for eventsByTag in the jdbc plugin? I see that it's ordered by creation date? Is that a timestamp assigned by the database? Can two events have the exact same timestamp and in that case will the order always be the same between different queries, i.e. can you use the offset as an unique position in the event stream?